openfecli/commands/plan_rhfe_network.py

```diff
@@ -111,12 +111,13 @@ def plan_rhfe_network_main(
 )
 @print_duration
 def plan_rhfe_network(molecules: List[str], yaml_settings: str, output_dir: str, n_cores: int, overwrite_charges: bool, n_protocol_repeats: int):
+    # TODO: make this match the rbfe network docstring, or vice-versa?
```

Comment:
this whole docstring deviates quite a bit from the rbfe docstring - do we want these to be more parallel in structure?

Reply:
Yes that would be good.

Reply:
will address in a follow-up PR https://github.com/OpenFreeEnergy/openfe/issues/1345